nginx/datadog_checks/nginx/nginx.py

```diff
@@ -70,21 +73,22 @@ def check(self, _):
         for row in metrics:
             try:
                 name, value, tags, metric_type = row
+                _tags = tags + ['host:%s' % self.hostname_from_url, 'port:%s' % self.port_from_url]
```

Comment:
In Python as a convention only unused variables have a single `_` prefix. It's a more verbose version of `_` as the variable name. Since here we are using the variable, I would suggest renaming it to `complete_tags`. Alternatively, rename what's currently named `tags` to `row_tags`, then you can name `_tags` as `tags`.